tests/phpunit/tests/dependencies/styles.php

```diff
@@ -74,6 +74,31 @@ public function test_wp_enqueue_style() {
 		$this->assertSame( '', get_echo( 'wp_print_styles' ) );
 	}
 
+	/**
+	 * Test async param
+	 *
+	 * @ticket #120 performance
+	 */
+	public function test_wp_enqueue_style_async() {
```

Comment:
You will get pushback with this test
You need to use a dataset  as we can't tell which call failed
I would also use a regex to look for the added code
https://www.loom.com/share/d9cb0508df284b28b8b67d8ae9acfa64

Reply:
I had a play and recorded a loom :-)